R/batch.R

```diff
@@ -16,8 +16,10 @@
 batch <- function(batch_fn, keys, splitting_strategy = NULL,
   combination_strategy = batchman::combine, size = 50, verbose = TRUE,
   trycatch = FALSE, stop = FALSE) {
-    if (is.batched_fn(batch_fn)) return(batch_fn)
+    if (isTRUE(attr(batch_fn, 'batched'))) return(batch_fn)
```

Comment:
Managed to figure this out.  I was calling `batch` in `testing-utils.R`, which apparently doesn't work.  I guess it tries to load prior to exporting happening.